server/routes/api/notifications.js

```diff
@@ -0,0 +1,46 @@
+const { Router } = require("express");
+
+const Notification = require("../../models/Notification");
+
+const Socket = require("../../services/socket");
+
+const router = Router();
+
+// get all notifications
+router.get("/all", async (req, res) => {
```

Comment:
I don't think we need this route (dont want people to be able to get all notifications of all users)